src/core.js

```diff
@@ -272,6 +272,16 @@ $.extend( $.validator, {
 				this.element( element );
 			}
 		},
+		oninput: function( element ) {
+			if ( element.name in this.submitted || element.name in this.invalid ) {
+				this.element( element );
+			} else {
+				return;
+			}
+		},
+
+		// DEPRECATED
+		// Please use `oninput` instead.
 		onkeyup: function( element, event ) {
```

Comment:
> I'll send a PR to document the oninput handler asap, and sure I'll mention this.

First we should make sure the implementations works as expected in all supported browsers without side-effects. The docs can be done later on.